projects/systelab-components/src/lib/datepicker/datepicker-time.component.html

```diff
@@ -19,14 +19,14 @@
 
     </ng-container>
 
-    <systelab-spinner class="d-flex h-100" id="hours" style="width: 81px;"
+    <systelab-spinner class="d-flex h-100" id="hours"
```

Comment:
You have removed the style, but I don't see the new class assigned

Reply:
The css is by selector.